frontend/app/views/spree/orders/show.html.erb

```diff
@@ -9,7 +9,9 @@
 
 <fieldset id="order_summary" data-hook>
   <legend><%= Spree.t(:order_number, number: @order.number) %></legend>
-  <h1><%= accurate_title %></h1>
```

Comment:
@Qwertie- it should be the other way around, we want the `accurate_title` to be displayed

Reply:
Ah ok, should I just remove <legend> then?